signup/index.html

```diff
@@ -0,0 +1,59 @@
+<html lang="ko">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>판다마켓</title>
+		<link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+		<link rel="stylesheet" href="/src/css/reset.css" />
+		<link rel="stylesheet" href="/src/css/global.css" />
+		<link rel="stylesheet" href="/src/css/login.css" />
+	</head>
+	<body>
+		<div class="logo">
+			<a href="/">
+				<img src="../src/img/common/logo.png" alt="판다마켓" width="396" />
```

Comment:
```suggestion
        <img src="../src/img/common/logo.png" alt="판다마켓"  />
```
img 태그에서 width를 주는 것보다 별도의 css로 너비를 조절하는 방식을 추천드립니다.
현재는 css 관련 살펴보아야 할 포인트가 두 군데입니다. 또한 개별 img 태그에 적용 된 width 값을 수정하려면 일일이 모든 html 파일을 찾아 값을 수정해야 합니다. 반면에 공통 css 파일로 속성을 관리하면 한 번의 수정으로 여러 군데에 있는 값을 수정할 수 있어서요!